### ISSUE TITLE
show Featured Campaigns on Explore page

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#5698]
Currently, the Explore page (/explore) lists the most recently created Campaigns. On Wiki Education Dashboard, we create many campaigns that have specific uses, sometimes well ahead of when they will become active, so the list of campaigns displayed isn't actually very useful for browsing and navigating to active current campaigns.

We'd like some way of controlling which campaigns to show instead of the current behavior of just showing the 10 most recent.

This could be implemented as a Setting record that holds an array of campaign slugs, and if present, the explore page would load those campaigns instead of the most recent ones. This could be controlled from a form on the Settings page (/settings).

Ideally, we would rework the Settings page so that it is accessible to all admins, not just super-admins, with the existing features limited to super-admins but this new one also accessible to regular admins.

**Comment by ujjwalpathaak** [comment]
Working on this

**Comment by ujjwalpathaak** [comment]
Hi @ragesoss I tried solving this issue.
I divided the filter system into 4 cateogries - Active, Past, Upcoming, Recent (Default)
Added a drop down in `/settings`
The filter is getting saved in the `settings` table like
![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/92005669/5552ac79-9a20-4e54-b6f0-f98ee9aadaa9)

Here is the basic functionality (video) - 
[Screencast from 18-03-24 04:40:41 PM IST.webm](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/92005669/3a86a7c9-649e-40b3-bec9-6bdedac93a4a)

Should I change how the filtering system is working?
If this looks like what is needed should I open a PR and continue working on this issue with the same implementation?
